typing/ctype.ml

```diff
@@ -7196,7 +7196,12 @@ let print_global_state fmt global_state =
    can't be written in Jkind. It's possible that, after jkind.ml is broken up,
    this problem goes away, because the dependency from Env to Jkind is pretty
    minimal. *)
-let type_equal env ty1 ty2 = is_equal env false [ty1] [ty2]
+let type_equal env ty1 ty2 =
```

Comment:
I'm suspicious of this change, since it affects everything, not just jkind normalization, and so is potentially breaking. Perhaps we should only do this inside `Jkind.normalize`?

Reply:
agreed

Reply:
No, I think that function is used only around jkind stuff. Look at the comment. I'm a little suspicious, too, but I think it should be fine.

Reply:
I just confirmed that it's used only for jkind stuff.

Reply:
Please Resolve if that satisfies you...